package.json

```diff
@@ -77,6 +77,7 @@
     "@rainbow-me/react-native-payments": "1.1.5",
     "@react-native-community/async-storage": "1.12.1",
     "@react-native-community/blur": "3.6.0",
+    "@react-native-community/cameraroll": "^4.1.2",
```

Comment:
hey @brunobar79 do we have a pattern now of locking versions we want to apply for all new dependencies